ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/CveStatusTabNavigation.tsx

```diff
@@ -1,23 +1,15 @@
 import React from 'react';
```

Comment:
honestly i wouldn't be opposed to getting rid of this component completely if we're deleting tab code.

Reply:
Ah good call - I'll move this code up to the parent component before I merge.